index.html

```diff
@@ -40,10 +40,11 @@ <h2>Security</h2>
 
 <h2>Privacy Protections Tests</h2>
 <ul>
-	<li><a href="./privacy-protections/request-blocking/">Request blocking</a></li>
-	<li><a href='./privacy-protections/fingerprinting/'>Fingerprinting</a></li>
-	<li><a href='./privacy-protections/storage-blocking/'>Storage blocking</a></li>
-	<li><a href='./privacy-protections/referrer-trimming/'>Referrer trimming</a></li>
+	<li><a href='https://privacy-test-pages.glitch.me/privacy-protections/request-blocking/'>Request blocking</a></li>
+	<li><a href='https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/'>Fingerprinting</a></li>
+	<li><a href='https://privacy-test-pages.glitch.me/privacy-protections/storage-blocking/'>Storage blocking</a></li>
+	<li><a href='https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/'>Referrer trimming</a></li>
+	<li><a href='http://privacy-test-pages.glitch.me/privacy-protections/https-upgrades/'>HTTPS upgrades</a></li>
```

Comment:
Some of those pages don't really work on github pages, which might be confusing, so I hardcoded full glitch urls.